net-analyzer/goaccess/goaccess-1.2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A real-time web log analyzer and interactive viewer that runs in a terminal"
+HOMEPAGE="http://goaccess.io"
+SRC_URI="http://tar.goaccess.io/${P}.tar.gz"
```

Comment:
"https://" works too (for both URLs)

Reply:
I just want to tell that:
* to get openssl for goaccess user should use "-libressl ssl"
* to get libressl for goaccess user should use "libressl ssl"
* to disable ssl support at all user should use "-ssl".

It is not clear for me ( as a user ) for what exactly "ssl" USE-flag is used in this ebuild. Does it just enable a general SSL support or it just enable an openssl support?

A user can run into a situation when he wants a libressl support and he will not get any SSL support because libressl is enabled and ssl is disabled.
I added "libressl? ( ssl )" to prevent this situation.
If there is a better way I'd like to know it. Thanks.